platform/features/hyperlink/bundle.js

```diff
@@ -0,0 +1,133 @@
+        "description": "Insert a hyperlink to reference a link",
+        "extensions": {
+        "types": [
+            {
+                "key": "hyperlink",
+                "name": "Hyperlink",
+                "cssClass": "icon-page",
+                "description": "A hyperlink to redirect to a different link",
+                "features": ["creation"],
+                "properties": [
+                  {
+                      "key": "url",
+                      "name": "URL",
+                      "control": "textfield",
+                      "pattern": "^(ftp|https?)\\:\\/\\/",
+                      "required": true,
+                      "cssClass": "l-input-lg"
+                  },
+                  {
+                      "key": "displayText",
+                      "name": "Text to Display",
+                      "control": "textfield",
+                      "required": true,
+                      "cssClass": "l-input-lg"
+                  },
+                  {
+                      "key": "showTitle",
+                      "control": "checkbox",
+                      "value": false
+                  },{
+                      "key": "displayFormat",
+                      "name": "Display Format",
+                      "control": "composite",
+                      "items": [
+                          {
+                              "control": "select",
+                              "options": [
+                                  {
+                                      "name": "Link",
+                                      "value": "link"
+                                  },
+                                  {
+                                      "value": "button",
+                                      "name": "Button"
+                                  }
+                              ],
+                              "cssClass": "l-inline"
+                          }
+                      ]
+                  },
+                  {
+                      "key": "openNewTab",
+                      "name": "Tab to Open Hyperlink",
+                      "control": "composite",
+                      "items": [
+                          {
+                              "control": "select",
+                              "options": [
+                                  {
+                                      "name": "Open in this tab",
+                                      "value": "thisTab"
+                                  },
+                                  {
+                                      "value": "newTab",
+                                      "name": "Open in a new tab"
+                                  }
+                              ],
+                              "cssClass": "l-inline"
+                          }
+                      ]
+                  }
+                ],
+                "model": {
+                    "displayFormat": ["link"],
```

Comment:
When the above aren't composite controls, this would just be `"link"` instead of `["link"]`.